packages/utilities/fast-gestures/src/utilities/get-coordinate-type.ts

```diff
@@ -0,0 +1,22 @@
+import { Point } from "../base/point";
+import { PointerCoordinateType } from "../pointer-coordinate-type";
+
+export function getCoordinateType(
```

Comment:
is this name accurate? This fn reads more like it's getting a point *by* coordinate type, is that correct?